libs/sdk-bindings/bindings-swift/Sources/BreezSDKNotification/SDKNotificationService.swift

```diff
@@ -0,0 +1,97 @@
+import UserNotifications
+import XCGLogger
+import os.log
+
+open class SDKNotificationService: UNNotificationServiceExtension {
+    var breezSDK: BlockingBreezServices?
+    var contentHandler: ((UNNotificationContent) -> Void)?
+    var bestAttemptContent: UNMutableNotificationContent?
+    var currentTask: TaskProtocol?
+    
+    var logger = {
```

Comment:
How does the user configure the logging directory?

Reply:
By overriding the init() then calling logger.setup with the extension log file path
https://github.com/breez/c-breez/blob/68a6cdd61b60c5ae000b4d4a5b9d581fd2dbb0ae/ios/Breez%20Notification%20Service%20Extension/NotificationService.swift#L10-L17

Is there a better approach @roeierez?

Reply:
What bothers me a bit is That apart from us being very opinionated about the logging frameworks the logging approach here and in android is different then the logging approach at the rust part of the sdk.
In the rust part the user provides a logger implementation (listener) and the app processes them the way it wants.
I wonder if we want to keep this approach also for kotlin and swift here.
Do you think passing a Logger interface implementation by the user will work? Perhaps another function that the user can override in the service/extension? We can also use the same SDKLogListener that we already have eliminate the need to introduce another interface.
We can still keep these implementations for users that want to use them out of the box.
I think If we do that we can allow ourselves to be more opinionated about the default implementation since the user has a way to override.

Reply:
I think its a pretty good idea. I would actually lean towards removing the implementations from the plugins, have c-breez as a reference logging implementation. It simplifies the plugins/dependencies and allows use to be less opinionated.